packages/streaming-image-volume-loader/src/StreamingImageVolume.ts

```diff
@@ -415,12 +414,14 @@ export default class StreamingImageVolume extends ImageVolume {
       const options = {
         // WADO Image Loader
         targetBuffer: {
-          arrayBuffer,
+          arrayBuffer:
+            arrayBuffer instanceof ArrayBuffer ? undefined : arrayBuffer,
           offset: imageIdIndex * lengthInBytes,
           length,
           type,
         },
         skipCreateImage: true,
+        transferPixelData: arrayBuffer instanceof ArrayBuffer,
```

Comment:
I was about to fix this in another PR. Do you need it here? since I prefer it to be a separate `fix: broken transferPixelData` for both volume and stack viewport. 

Does it hurt to set `transferPixelData` to true for SharedArrayBuffer too? although I guess it is shared by construct ..?

Reply:
I can remove this for your fix. It shouldn’t cause any issue to copy rather than transfer the encoded pixel array data